app/models/course/video/submission.rb

```diff
@@ -14,6 +14,7 @@ class Course::Video::Submission < ApplicationRecord
   validates :video, presence: true
 
   belongs_to :video, inverse_of: :submissions
+  belongs_to :creator, class_name: User.name, inverse_of: :video_submissions
```

Comment:
`Course::Assessment::Submission`s uses scopes `by_user` and `by_users` to find submissions created by a given user.

Can we also do the same thing for video submissions so it's consistent?